File: src/main/java/com/github/frcsty/frozenactions/actions/player/JsonMessageAction.java
package com.github.frcsty.frozenactions.actions.player;

import com.github.frcsty.frozenactions.actions.Action;
import com.github.frcsty.frozenactions.actions.ActionContext;
import org.bukkit.Bukkit;
import org.jetbrains.annotations.NotNull;

public final class JsonMessageAction implements Action {

    @Override
    public @NotNull String getId() {
        return "JSONMESSAGE";
    }

    @Override
    public void run(final @NotNull ActionContext context) {
        Bukkit.dispatchCommand(Bukkit.getConsoleSender(), "tellraw " + context.getPlayer().getName() + " " + context.getData());
    }
}

File: src/main/kotlin/com/github/frcsty/frozenactions/actions/player/JsonMessageAction.kt
package com.github.frcsty.frozenactions.actions.player

import com.github.frcsty.frozenactions.actions.Action
import org.bukkit.Bukkit
import org.bukkit.entity.Player

object JsonMessageAction : Action {

    override val id = "JSONMESSAGE"

    override fun run(player: Player, data: String) {
        Bukkit.dispatchCommand(Bukkit.getConsoleSender(), "tellraw ${player.name} $data")
    }
}